README.md

```diff
@@ -206,6 +206,39 @@ int main(void){
 }
 ```
 
+## Run-time trace group filtering
+
+You can control run-time the tracing per trace group. Each of your modules should should define the trace group (using `#define TRACE_GROUP "abcd"` in the .c/.cpp file) and you can use this trace group for controlling the tracing.
```

Comment:
> Each of your modules should should define the trace group (using `#define TRACE_GROUP "abcd"` in the .c/.cpp file)

This is always the case if using the trace macros.  If it's not mentioned anywhere it should perhaps be mentioned at a more generic location.

Reply:
It is actually mentioned in the pre-requisites. I've re-worded this a bit, but wanted a reminder here...